image/CMakeLists.txt

```diff
@@ -9,7 +9,11 @@ add_assets_target(
     SOURCE_DIR ${ASSETS_SOURCE_DIR}
     DEST_DIR ${ASSETS_DEST_DIR}
     DEVEL ${WITH_DEVELOPMENT_FEATURES}
-    DEPENDS json-target json-images-target
+    DEPENDS
+        json-common-target
+        "$<$<STREQUAL:${ASSETS},Proprietary>:json-proprietary-target>"
```

Comment:
please rename to `ASSETS_TYPE` for example, just `ASSETS` seems to ambiguous

Reply:
renamed